control_msgs/action/AntipodalGripperCommand.action

```diff
@@ -0,0 +1,11 @@
+sensor_msgs/JointState command # The velocity and effort fields are optionally and not used if empty.
+---
```

Comment:
Suggest adding an option to the request to change the `allow_stalling` variable.

Reply:
Adding this could be annoying because it would require users to always fill it in if they want stalling to always be allowed so feel free to ignore this suggestion.